### PR TITLE
Doc fix: add info about UUID

### PR DESCRIPTION
**Pull request by ogorbacheva**
I hereby agree to the terms of the CLA available at: https://yandex.ru/legal/cla/?lang=en

Category:
- Doc fixes

Task:
- DOCAPI-5125

### CODE REVIEW

File: docs/en/query_language/functions/uuid_function.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# generateUUIDv4 function
+
+Generates [UUID](../../data_types/uuid.md) of version 4.
```
Comment (blinkov) [inline docs/en/query_language/functions/uuid_function.md]: "version 4" could have been a link to https://tools.ietf.org/html/rfc4122#section-4.4

File: docs/en/data_types/uuid.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
+# UUID {#uuid-data-type}
+
+Universally unique identifier (UUID) is a 16-byte number used to identify the records. For detailed information about the UUID, see [Wikipedia](https://en.wikipedia.org/wiki/Universally_unique_identifier).
+
+The UUID format is represented below:
```
Comment (blinkov) [inline docs/en/data_types/uuid.md]: Actually it's just a human readable representation, physically it's 16 bytes.

File: docs/toc_en.yml [hunk 3]
```diff
@@ -80,6 +81,7 @@ nav:
     - 'Bit': 'query_language/functions/bit_functions.md'
     - 'Hash': 'query_language/functions/hash_functions.md'
     - 'Generating Pseudo-Random Numbers': 'query_language/functions/random_functions.md'
+    - 'generateUUIDv4': 'query_language/functions/uuid_function.md'
```
Comment (blinkov) [inline docs/toc_en.yml]: There are multiple UUID-related functions, so the article should be named like others in plural (uuid_functions.md) with corresponding title.

See:
```
SELECT *
FROM system.functions
WHERE name LIKE '%UUID%'
FORMAT TSV

generateUUIDv4	0	0
toUUID	0	0
UUIDStringToNum	0	0
dictGetUUID	0	0
dictGetUUIDOrDefault	0	0
UUIDNumToString	0	0

6 rows in set. Elapsed: 0.003 sec.
```

File: docs/toc_en.yml [hunk 4]
```diff
@@ -42,6 +42,7 @@ nav:
   - 'Date': 'data_types/date.md'
   - 'DateTime': 'data_types/datetime.md'
   - 'Enum': 'data_types/enum.md'
+  - 'UUID': 'data_types/uuid.md'
```
Comment (blinkov) [inline docs/toc_en.yml]: I'd put it higher in TOC right after FixedString, because it's somewhat related

**Review by blinkov — CHANGES_REQUESTED** [review]

**Review by blinkov — CHANGES_REQUESTED** [review]
Current patch contains a lot of unrelated changes + uuid_function.md is not renamed to plural

**Comment by ogorbacheva** [comment]
Yes, is in progress, tomorrow i ll do PR

File: docs/ru/query_language/misc.md [hunk 5]
```diff
@@ -5,17 +5,50 @@
...
-Если таблица перед этим была отсоединена (``DETACH``), т.е. её структура известна, то можно использовать сокращенную форму записи без определения структуры.
+Если таблица перед этим была отсоединена (`DETACH`), т.е. её структура известна, можно использовать сокращенную форму записи без определения структуры.
 
 ```sql
 ATTACH TABLE [IF NOT EXISTS] [db.]name [ON CLUSTER cluster]
 ```
 
 Этот запрос используется при старте сервера. Сервер хранит метаданные таблиц в виде файлов с запросами `ATTACH`, которые он просто исполняет при запуске (за исключением системных таблиц, которые явно создаются на сервере).
 
+## CHECK TABLE
```
Comment (blinkov) [inline docs/ru/query_language/misc.md]: This is from other patch that was already merged to master, looks like you haven't done `git pull`.

File: docs/en/query_language/functions/uuid_function.md [hunk 6]
```diff
@@ -56,14 +63,42 @@ Code: 376. DB::Exception: Received from clickhouse-server:9000, 172.17.0.2. DB::
...
+└───────────┘
+```
+
 ## UUIDNumToString
 
+Accepts a FixedString(16) value. Returns a string containing 36 characters in text format.
+
+**Returned value**
+
 ## dictGetUUID
```
Comment (blinkov) [inline docs/en/query_language/functions/uuid_function.md]: I'm not sure if it's worth duplicating these functions here, there's a separate section with functions related to dictionaries and for other data types those are documented only once.

File: docs/en/query_language/functions/uuid_function.md [hunk 7]
```diff
@@ -43,7 +50,7 @@ UUID.
 └──────────────────────────────────────┘
 ```
 
-Note that the String value has to correspond to the UUID format.
+The `toUUID()` function returns an error if the String value does not correspond to the UUID format. For example:
 
 ``` sql
 :) SELECT toUUID('61f0c4045cb3-11e7907ba6006ad3dba0') AS uuid
```
Comment (blinkov) [inline docs/en/query_language/functions/uuid_function.md]: this example leads to `DB::Exception: Cannot parse uuid 61f0c4045cb3-11e7907ba6006ad3dba0: Cannot parse UUID from String.`

**Review by blinkov — CHANGES_REQUESTED** [review]
+ the file is still not renamed to uuid_function**s**.md

**Pull request closed without merging**